examples/angularjs_require/js/directives/todoEscape.js

```diff
@@ -0,0 +1,16 @@
+/*global define*/
+define(['app'], function (app) {
+	'use strict';
+
+	app.directive('todoEscape', function () {
+		var ESCAPE_KEY = 27;
+
+		return function (scope, elem, attrs) {
+			elem.bind('keydown', function (event) {
+				if (event.keyCode === ESCAPE_KEY) {
```

Comment:
I honestly have no idea if it is taking care of that. cc'ing @passy as this was mostly copy-pasta from https://github.com/tastejs/todomvc/commit/2f6242683818611a585ff502671db08c5af06d6f.